pkgs/development/python-modules/dnspython/default.nix

```diff
@@ -24,10 +26,14 @@ buildPythonPackage rec {
   disabledTests = [
     # dns.exception.SyntaxError: protocol not found
     "test_misc_good_WKS_text"
+    # fails if IPv6 isn't available
+    "test_resolver_override"
   ];
 
   nativeBuildInputs = [
     setuptools-scm
+  ] ++ lib.optional stdenv.isDarwin [
+    cacert
```

Comment:
are these changes necessary to build on Darwin?

Reply:
Yes, at least on my machine, I get failing tests saying, e.g.:
```
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)
```

I just force pushed that adds it to `checkInputs`, instead of a `nativeBuildInputs`.

Reply:
I also see the same error on hydra.